perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ServicesManagerMethod.java

```diff
@@ -479,7 +479,7 @@ public Service call(ApiCaller ac, Deserializer parms) throws PerunException {
 	 |        +-------Attrs
 	 +...
 	 </pre>
-	 *
+	 * @deprecated - use getHashedHierarchicalData
```

Comment:
I'm not sure if our docs parser won't go crazy about it, since it already skips all that ``<pre>`` part above and doesnẗ handle more lines of param description.

Also it shows deprecation message on hover, so it will be like ``- use getHashedHierarchicalData``. I would suggest to to remove leading dash.

Reply:
Fixed.